capstone/about.html

```diff
@@ -0,0 +1,130 @@
+      </div>
+    </nav>
+    <div class="menu">
+      <a class="menu-button open" href="#"><i class="fa-solid fa-bars"></i></a>
+      <a class="menu-button close" href="#"><i class="fa-solid fa-xmark"></i></a>
+      <div hidden class="menu-content">
+        <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+        <div class="menu-items">
+          <a class="aboutPage" href="about.html">About</a>
+          <a href="home.html#program">Program</a>
+          <a href="home.html#join">Join</a>
+          <a href="about.html#sponsor">Sponsor</a>
+          <a href="about.html#news">News</a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="main">
+      <div class="homeContent about">
+        <h2 class="greet">"Hello! Rock & Roll Fans"</h2>
+        <div class="holder">
+          <h1 class="title">ROCK & ROLL HALL OF FAME 2023</h1>
+          <div class="underLine"></div>
+        </div>
+        <p class="description">
+          <span>Rock and Roll Hall of Fame and Museum, museum and hall of fame in Cleveland that celebrates the history and cultural significance of rock music and honours the contributions of those who have played an important role in the music's creation and dissemination.</span>
+          <span class="deskAboutDesc">To be eligible for nomination, an individual artist or band must have released its first commercial recording at least 25 years prior to the year of nomination. Eight out of 14 of the Nominees are on the ballot for the first time, including Sheryl Crow, Missy Elliott, Joy Division/New Order, Cyndi Lauper, George Michael, Willie Nelson, The White Stripes, and Warren Zevon. This is the first year of eligibility for Missy Elliott and The White Stripes.</span>
+        </p>
+        <p class="contact">Please contact us per Email for any furter question about R&R Hall of Fame 2023<br><span class="email">r&rhallfame2023@rrhof.org</span></p>
+      </div>
+    </div>
+    <div class="creativeLogo">
+      <div class="holder">
+        <h2 class="title">Creative Commons R&R Hall of Fame 2023 Logo</h2>
+        <div class="underLine"></div>
+      </div>
+      <p class="logoDesc">
+        <span>The logo of R&R Hall of Fame 2023 was decided through the logo competition from 4, April to 5, May</span>
+        <span>reflects the diverse artists and music that the Rock & Roll Hall of Fame honors and celebrates. These artists have created their own sounds that have impacted generations and influenced countless others that have followed in their footsteps.</span>
+      </p> 
+      <img class="aboutLogo" src="images/logo.svg">
+    </div>
+    <div id="news" class="pastEvents">
+      <div class="titleWrap">
+        <div class="holder">
+          <h2 class="title">See the past R&R Hall of Fame Events</h2>
+          <div class="underLine"></div>
+        </div>
+        <p class="description">Take a look at the last two R&R Hall of Fame Events</p>
+      </div>
```

Comment:
- [ ] kindly try to use the correct color of the Behance here in the text of the about page and with that the project will be similar to the Behance and nice 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/217928415-bd504f81-b011-4d3f-8076-a532d2427e34.png)    |    ![image](https://user-images.githubusercontent.com/83360636/217928344-1a27ca63-01eb-4865-860c-2746caf2b251.png)   |